src/parser/cvc/Cvc.g

```diff
@@ -510,12 +509,12 @@ api::Term addNots(api::Solver* s, size_t n, api::Term e) {
 
 @header {
 /**
- ** This file is part of the CVC4 project.
- ** Copyright (c) 2009-2016 by the authors listed in the file AUTHORS
- ** in the top-level source directory) and their institutional affiliations.
- ** All rights reserved.  See the file COPYING in the top-level source
- ** directory for licensing information.
- **/
+ * This file is part of the CVC4 project.
+ * Copyright (c) 2009-2016 by the authors listed in the file AUTHORS
+ * in the top-level source directory) and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ */
```

Comment:
This has not been updated for a long time...

Reply:
Hm, this is weird. I ran the script on everything. It seems the update is only triggered if there is an update to the top contributors, which makes sense, I guess. I'll update manually.

Reply:
This is already updated.